src/libexpr/eval.cc

```diff
@@ -3068,6 +3069,11 @@ SourcePath EvalState::findFile(const LookupPath & lookupPath, const std::string_
 
         auto res = (r / CanonPath(suffix)).resolveSymlinks();
         if (res.pathExists()) return res;
+
+        // Backward compatibility hack: throw an exception if access
+        // to this path is not allowed.
```

Comment:
Why is this a hack?

Reply:
Yeah where is `FilteringSourceAccessor` even being used anymore? I only see it used with `GitExportIgnoreSourceAccessor`